### PR TITLE
PBI 1 register

### PR DESCRIPTION
**Pull request by Zsaschz**
- created helpers for calling api
- implemented register frontend
- change test configuration

### CODE REVIEW

File: src/Routes/web.php [hunk 1]
```diff
@@ -11,4 +11,5 @@
     ->middleware(['web', HandleInertiaRequests::class])
     ->group(function () {
         Route::get('/', [InertiaController::class, 'home']);
+        Route::get('/register', [InertiaController::class, 'register']);
     });
```
Comment (kanziebub) [inline src/Routes/web.php:15]: newline uwu

**Review by kanziebub — APPROVED** [review]
check your tests and don't forget prev comments, approve for now.

File: src/Routes/web.php (hunk 1 above)
Comment (AldiNFitrah) [inline src/Routes/web.php]: newline

File: tests/unit/register.spec.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
+import { mount } from "@vue/test-utils";
+import Register from "../../src/resources/app/pages/register.vue";
+
+describe("Register", () => {
+
+  describe("when loaded", () => {
+    const wrapper = mount(Register, {
+    });
```
Comment (AldiNFitrah) [inline tests/unit/register.spec.js:10]: why not putting them in the same line?

**Review by AldiNFitrah — COMMENTED** [review]
please do re-request after the issues are solved

File: src/resources/api/auth.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+        let timeNow = date.getTime();
+        localStorage.setItem(
+          window.btoa("tokenAccessTime"),
+          window.btoa(timeNow)
+        );
+      }
+    });
+    return response;
+  },
+};
```
Comment (AldiNFitrah) [inline src/resources/api/auth.js:23]: nit: need some whitespaces to be able to "breathe"
nit label means you're free to not implement it, just resolve the issue if you prefer to do so

File: src/resources/api/utils/handleError.js [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+      });
+      return Promise.reject(data);
+    } else if (status === 503) {
+      let data = error.response.data;
+      let message = error.response.data.message;
+      data.status = status;
+      data.message = message;
+      return Promise.reject(data);
+    }
+  }
```
Comment (AldiNFitrah) [inline src/resources/api/utils/handleError.js:29]: nit: whitespaces

**Review by AldiNFitrah — APPROVED** [review]

**Pull request merged**